### DISCUSSION TITLE
Document ways to help fund textual

### DISCUSSION THREAD

**Discussion by UmBsublime** [Textualize/textual#1918] [General]
I was looking for means to contribute financially to the Textualize org, but
sifting through the docs I didn't find any mention of this.
I did however find something in the [git history](https://github.com/Textualize/textual/commits/v0.12.1/.github/FUNDING.yml) of textual related to github actions.

It seems to point to https://ko-fi.com/textualize which I believe is up to date.

In addition to github actions, it would be neat to find a reference to this on https://textual.textualize.io/

Q1: Is https://ko-fi.com/textualize still the up to date way to contribute ?
Q2: Do you think it's worth mentioning somewhere (README/textualize.io/gh-actions)?

**Comment by willmcgugan** [comment]
It's a nice thought. But we're funded! There is currently five of us working full time.

I've been meaning to shutdown the ko-fi account and donate that money to charity.

If you would like to support Textual, the best way would be to spread the word. Let people know about it. And of course, bugs, suggestions, PRs etc.
